packages/babel-preset-es2015/test/fixtures/traceur/Classes/DefaultConstructor.js

```diff
@@ -18,9 +18,8 @@ assert.isFalse(
 assert.isFalse(
     Object.getOwnPropertyDescriptor(C.prototype, 'constructor').enumerable);
 
-// Ensure that we don't try to call super() in the default constructor.
 class D extends null {}
-var d = new D();
+assert.throws(() => new D());
```

Comment:
I wonder what happened with TC39's proposal to actually make `extends null` work.

As it is implemented/specified right now, `extends null` is explicitly allowed as a special case, but it's impossible to actually construct such a class.